File: app/src/main/java/com/foodie/foodvisit/FoodVisitApp.java
package com.foodie.foodvisit;

import android.app.Application;
import android.content.Context;

/**
 * Created by kprabhu on 11/12/17.
 */

public class FoodVisitApp extends Application{
    private static FoodVisitApp context;

    @Override
    public void onCreate() {
        super.onCreate();
        context = (FoodVisitApp) getApplicationContext();
    }

    public static Context getAppContext() {
        return context;
    }

}

File: app/src/main/java/com/mintanable/foodvisit/FoodVisitApp.kt
package com.mintanable.foodvisit

import android.app.Application
import com.google.android.libraries.places.api.Places
import dagger.hilt.android.HiltAndroidApp

@HiltAndroidApp
class FoodVisitApp : Application() {
    override fun onCreate() {
        super.onCreate()
        if (!Places.isInitialized()) {
            Places.initialize(applicationContext, BuildConfig.GOOGLE_MAPS_KEY)
        }
    }
}
